samples/ServerApplication/Program.cs

```diff
@@ -24,26 +27,36 @@ public static async Task Main(string[] args)
 
             services.AddSignalR();
 
+            services.TryAddSingleton<ObjectPoolProvider, DefaultObjectPoolProvider>();
```

Comment:
This feels like an unnecessary dependency

Reply:
You mean using the String builder pool?

Reply:
Yes

Reply:
The `LoggingStream` creates two new `StringBuilder`s for every message that it logs. Why wouldn't you optimized that?